anyway/vehicle_type.py

```diff
@@ -2,7 +2,11 @@
 from typing import List, Union
 import logging
 import math
-from flask_babel import _
+
+try:
+    from flask_babel import _
+except ModuleNotFoundError:
+    pass
```

Comment:
This is used from two methods. It's probably better to move the import to inside those methods (temporarily, with a comment explaining why it's there).

Reply:
to be handled in #1876